tests/bootstrap_tests/test_resolver.py

```diff
@@ -9,13 +9,10 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 import os
-import mock
 from mock import patch
-import tempfile
-import uuid
 import sgtk
 
-from tank_test.tank_test_base import *
+from tank_test.tank_test_base import setUpModule, TankTestBase # noqa
```

Comment:
Tells PEP8 to not complain about setUpModule not actually being used. We have to import setUpModule because the unittest framework will look for it, but we don't actually use it. So now the file is warning free. :)